councilmatic_core/management/commands/convert_rtf.py

```diff
@@ -15,6 +15,7 @@
 from councilmatic_core.models import Bill
 
 logging.config.dictConfig(settings.LOGGING)
+logging.getLogger("requests").setLevel(logging.WARNING)
```

Comment:
Was that logger logging too much?

Reply:
Yeah - the python Logging module captured all INFO messages from the requests library (which made for noisy logs).